### PR TITLE
fix giou loss test

### PR DESCRIPTION
**Pull request by fsx950223**
Related #1320.
As [Reduction.NONE](https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/keras/losses/Reduction) has changed. Change reduction mode to Reduction.SUM_OVER_BATCH_SIZE for unifying graph and eager reduction behavior.

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Thank you for the pull request!
